plugins/nessus.rb

```diff
@@ -410,13 +479,19 @@ def cmd_nessus_server_status(*args)
       end
 
       def cmd_nessus_admin(*args)
+        while (arg = args.shift)
+          case arg
+          when '-h', '--help'
+            print_status("nessus_admin")
+            print_status("Example:> nessus_admin")
+            print_status()
+            print_status("Checks to see if the current user is an admin")
+            print_status("Use nessus_user_list to list all users")
+            return
+          end
+        end
         if args[0] == "-h"
```

Comment:
I think the original author placed that there for formatting. Void-in: can
I nix the formatting line(s)?
On Feb 3, 2016 11:22 AM, "James Lee" <notifications@github.com> wrote:

> In plugins/nessus.rb
> <https://github.com/rapid7/metasploit-framework/pull/6517#discussion_r51744471>
> :
>
> > @@ -410,13 +479,19 @@ def cmd_nessus_server_status(*args)
> >        end
> >
> >        def cmd_nessus_admin(*args)
> > +        while (arg = args.shift)
> > +          case arg
> > +          when '-h', '--help'
> > +            print_status("nessus_admin")
> > +            print_status("Example:> nessus_admin")
> > +            print_status()
> > +            print_status("Checks to see if the current user is an admin")
> > +            print_status("Use nessus_user_list to list all users")
> > +            return
> > +          end
> > +        end
> >          if args[0] == "-h"
>
> Body is empty, delete this line.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rapid7/metasploit-framework/pull/6517/files#r51744471>
> .
>